Cargo.toml

```diff
@@ -123,9 +123,9 @@ subtle = "2.2.3"
 # - Warning: One of the downstream consumers requires that SQLite be available through
 #   CocoaPods, due to being bound to React Native. We need to ensure that the SQLite
 #   version required for `rusqlite` is a version that is available through CocoaPods.
```

Comment:
I checked and I believe our constraints here are met for all `rusqlite` versions existing in this PR:

rusqlite | Bundled SQLite | Matching CocoaPods sqlite3 tag
-----|--------|---
0.29 | 3.41.2 | :white_check_mark:
0.30 | 3.44.0 | :white_check_mark:
0.31 | 3.45.1 | :white_check_mark:
0.32 | 3.46.0 | :white_check_mark: